### DISCUSSION TITLE
港籍怎么润

### DISCUSSION THREAD

**Discussion by 3140184367linyan** [The-Run-Philosophy-Organization/run#361] [Q&A]
港籍文科生想往外润好润吗？往哪个方向比较好，预计内地本科研究生考到香港硕士留学有可能吗😭

**Comment by XutongJin** [comment]
对这些了解比较少，感觉港籍润英国会方便很多。
ps：港籍还需要润吗？

**Reply by 3140184367linyan** [reply]
我家在大陆没有在香港，本来觉得自己努努力回港就好，可是又感觉香港也越来越和内地一样了

**Comment by Doris324** [comment]
港籍可以看一下加拿大的救生艇计划，当地毕业/工作申请走这个计划就能润。
英国的话有BNO会好润一点。

**Reply by 3140184367linyan** [reply]
好滴，谢谢

**Comment by edchung777** [comment]
加拿大救生艇很好用，我女朋友就是这么留的，不要花钱而且基本看你是HK护照就给。她还是内地出生的香港永久居民，加拿大政府不管，直接给open work permit

**Reply by Doris324** [reply]
请问你女朋友当时在HK是黑印身份证还是绿印身份证呢？我在考虑要不要通过我家人先申请香港身份再去CA，但目前申请下来就只能先拿绿印，不知道绿印可不可以走救生艇计划

**Reply by 3140184367linyan** [reply]
好滴，非常感谢

**Reply by edchung777** [reply]
> 请问你女朋友当时在HK是黑印身份证还是绿印身份证呢？我在考虑要不要通过我家人先申请香港身份再去CA，但目前申请下来就只能先拿绿印，不知道绿印可不可以走救生艇计划

她是小时候直接居港的，现在拿的是完整的香港护照，不是身份书。身份书我感觉可能不行

**Reply by Doris324** [reply]
谢谢你，我知道啦
